pages/index.js

```diff
@@ -1,15 +1,45 @@
+import Link from "next/link.js";
+import styled from "styled-components";
 import useSWR from "swr";
 
+const List = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+  padding-left: 0;
+`;
+const ListItem = styled.li`
+  position: relative;
+  width: 100%;
+`;
+
 export default function Home() {
-  const { data: myrecipes } = useSWR("/api/myrecipes", { fallbackData: [] });
+  const { data } = useSWR("/api/recipes", { fallbackData: [] });
   return (
     <>
       <h1>Rezepte</h1>
-      <ul>
-        {myrecipes.map((myrecipe) => {
-          return <li key={myrecipe._id}>{myrecipe.title}</li>;
+      <List role="list">
+        {data.map((recipe) => {
+          return (
+            <ListItem key={recipe._id}>
+              {recipe.title}
+              <br></br>
+              {recipe.type}
+              <br></br>
+              {recipe.course}
+              <br></br>
+              {recipe.yield}
+              <br></br>
+              {recipe.time}
+            </ListItem>
+          );
         })}
-      </ul>
+      </List>
+      <Link href="/create" passHref legacyBehavior>
+        <Link>+ recipe</Link>
+      </Link>
```

Comment:
you don't need passHref or legacyBehavior here, this should do just fine:

```suggestion
     <Link href="/create">+ recipe</Link>
```